README.md

```diff
@@ -1 +1,77 @@
-# Javascript-capstone-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Javascript-capstone-project
+
+The JavaScript capstone project is about building a web application based on an external API. We selected an API that provides data about Pokemon and then build the webapp around it. The webapp has 2 user interfaces:
+- A home page showing a list of items that you can "like."
+- A popup window with more data about an item that you can comment on it or reserve it for a period of time.
+
+<div align="center">
+<img  src="https://user-images.githubusercontent.com/99416452/197285485-76fb9f85-e314-4fa9-8d16-d81ec0c97045.png">
+</div>
+
+
+## Built With
+
+- HTML
+- CSS
+- Javascript ES6
+- Jest
+
+
+## Presentation video
+
+[video Link](https://user-images.githubusercontent.com/99416452/197282376-50f5b22c-bb46-4ad0-9a54-95f2ff2d0e06.mp4)
+
+## Live Demo
+
+[Live Demo Link](https://kal2944.github.io/Javascript-capstone-project/)
+
+
+## To get a local copy run the following steps:
+- Open your terminal or command line
+- Run "git clone Paste [this link](https://github.com/kal2944/Javascript-capstone-project.git)"
+- Open the folder with your code editor
+- Create a branch to work on
+- Run `npm run build`
+- Run `npm start` to start the server
+- Or open the `index.html` file in the dist directory with liveserver
+
+## Authors
+
+👤 **Author1**
+
+👤 **Cesar Alberto Valencia Aguilar**
+
+- GitHub: (https://github.com/cvalencia1991)
+- Twitter: (https://twitter.com/cvalenciaguilar)
+- LinkedIn: (https://www.linkedin.com/in/cvalenciaguilar/)
+
+
+👤 **Author2**
+
+
+👤 **Kalkidan Daniel**
+
+- GitHub: [@kal2944](https://github.com/kal2944)
+- Twitter: [@twitterhandle](https://twitter.com/Kalkida01088197)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kalkidan-daniel-b2a204238/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Your link to your license file is broken. Kindly fix this link. This is your license file so please take it seriously 😊